### PR TITLE
Improve destructuring workout

### PR DESCRIPTION
**Pull request by kapnobatai137**
defaults-values-apply-only-to-undefined-and-not-null.md
- add JS tags to code snippets
- prettify code blocks
- improve first sentence
- improve RQ
- remove `ES6` from quiz question

destructuring-part-1.md
- remove `ES6` from practice question
- prettify code blocks

destructuring-part-3.md
- prettify code blocks

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md?ref=f6b3613a3e594b1a82da32c0496496656f638300)
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/destructuring/destructuring-part-1.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/destructuring/destructuring-part-1.md?ref=f6b3613a3e594b1a82da32c0496496656f638300)
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/destructuring/destructuring-part-3.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/destructuring/destructuring-part-3.md?ref=f6b3613a3e594b1a82da32c0496496656f638300)

File: javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md [hunk 1]
```diff
@@ -78,9 +86,7 @@ const {a = 1} = {a: undefined}
 ## Revision
 
 ```javascript
-function bar(a = 1) {return a}
-//foo(null) === null
-const {a = 1} = {a: null}
+const { a = 1 } = { a: null };
```
Comment (nem035) [inline javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md]: Weird that this revision question doesn't have text. Can we maybe copy the same text from the practice question?

Comment (kapnobatai137) [reply javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md]: done

Comment (kapnobatai137) [reply javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md]: done

**Pull request merged**